grammar/src/main.rs

```diff
@@ -0,0 +1,258 @@
+use std::{io::Read, process::exit};
+
+use colored::Colorize;
+use ebnf::{
+    io::MarkableReader,
+    parser::{Event, Parser},
+    Error, Location, Syntax,
+};
+use toml::{Table, Value};
+
+use crate::generate::generate;
+
+mod generate;
+
+fn main() {
+    // Parse EBNF
+    let mut definition_file = String::new();
+    let _file = std::fs::File::open("proforma.ebnf")
```

Comment:
Maybe this could be inlined during compilation with an `include_str!` macro so that the tester portable, or alternatively be made an argument to make it able to test new grammars?